### DISCUSSION TITLE
Script to "translate" force fields to tinker format

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker-gpu#211] [Q&A] [answered]
Dear all,

is anybody aware of any script that can reformat force fields from other programs (in particular GROMACS) to TINKER? I really like how tidy TINKER is, in comparison for instance GROMACS. Also, it would really help to be able to "jump" between fixed-charge and polarizable simulations easily.

Thanks!
- Jacek

**Comment by zhi-wang** [comment]
Dr. Ponder is more experienced with general questions like this one, and I'm curious about it too. I'll forward your question to him if he misses this one.

**Comment by jayponder** [comment, marked as answer]
Unfortunately, I do not know of any such script or program. There are so many small details that differ between the parameter files for different programs that doing this in general would be difficult. I have certainly done lots of this type of translation in the past, and have always just written small simple scripts specific to the particular translation being performed at the time.

The critical issue is always to make sure that you get exactly the same energy values when using the original program and parameters vs. the translated parameters in Tinker.

**Reply by jacekkozuch** [reply]
Thanks for the quick response. 
Good to know. Maybe I'll just give it a try at some point and see if I manage to do it for a newer parameter set from GROMACS.
